test/unit/code.js

```diff
@@ -0,0 +1 @@
+module.exports = "module.exports=function(a){function b(d){if(c[d])return c[d].exports;var e=c[d]={i:d,l:!1,exports:{}};return a[d].call(e.exports,e,e.exports,b),e.l=!0,e.exports}var c={};return b.m=a,b.c=c,b.i=function(d){return d},b.d=function(d,e,f){b.o(d,e)||Object.defineProperty(d,e,{configurable:!1,enumerable:!0,get:f})},b.n=function(d){var e=d&&d.__esModule?function(){return d['default']}:function(){return d};return b.d(e,'a',e),e},b.o=function(d,e){return Object.prototype.hasOwnProperty.call(d,e)},b.p='',b(b.s=0)}([function(a,b){'use strict';Object.defineProperty(b,'__esModule',{value:!0});b.data=function(f,g){var h={logo:{href:'',url:f.params.logoUrl,alt:f.params.title+' logo'},title:f.params.title,timezoneOffset:'?'};f.plugins.getTimezoneOffset&&(h.timezoneOffset=f.plugins.getTimezoneOffset()),g(null,h)}}]);";
```

Comment:
I think my preference would be to make it a component and bring it to `/test/fixtures/mocked-components/missing-plugin.js` (look at the others in that dir to follow same convention)